project-base/composer.json

```diff
@@ -19,7 +19,7 @@
         }
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.4.1",
```

Comment:
What is the reason of skipped 7.4.0? Am I missing any bug which was introduced?

Reply:
it's mainly because of roave/better-reflection package, so we can update this easy https://github.com/Roave/BetterReflection/blob/4.11.0/composer.json